pkg/backend/diy/backend.go

```diff
@@ -795,29 +799,66 @@ func (b *diyBackend) ListStacks(
 		return nil, nil, err
 	}
 
+	// Get the max parallel value from environment variable or use a default value
+	maxParallel := b.Env.GetInt(env.DIYBackendParallel)
+	if maxParallel <= 0 {
+		maxParallel = 10 // Default to 10 parallel operations if not specified
```

Comment:
Maybe add a `func (b *diyBackend) getParallel()` method so we don't have this magic 10 number show up twice.